src/main/groovy/com/f2prateek/javafmt/Formatter.groovy

```diff
@@ -0,0 +1,24 @@
+package com.f2prateek.javafmt
+
+import org.gradle.api.Project
+
+class Formatter {
+  def formatterClass
+  def formatter
+
+  Formatter(Project project) {
+    def config = project.configurations.getByName(AndroidJavafmtPlugin.CONFIG_NAME)
+    def urls = new ArrayList<URL>()
+    config.files.each { File file ->
+      urls += file.toURI().toURL()
+    }
+    def cl = new URLClassLoader(urls.toArray(new URL[0]));
+
+    formatterClass = cl.loadClass("com.google.googlejavaformat.java.Formatter")
```

Comment:
I guess what I meant was, why don't we link to a specific version, rather than using reflection?  Something similar to the "provided" scope for regular projects.